src/platform/platform_linux.cpp

```diff
@@ -189,14 +189,6 @@ std::string get_alias_script_path(const std::string& alias)
 
     return aliases_folder.absoluteFilePath(QString::fromStdString(alias)).toStdString();
 }
-
-std::filesystem::path multipass_final_storage_location()
-{
-    const auto user_specified_mp_storage = MP_PLATFORM.multipass_storage_location();
-    const auto mp_final_storage = user_specified_mp_storage.isEmpty() ? mp::utils::snap_common_dir()
```

Comment:
So, why was this always looking in the snap directory even if `MP_PLATFORM.multipass_storage_location` was set?